adapters/handlers/rest/configure_api.go

```diff
@@ -595,6 +595,10 @@ func parseVotersNames(cfg config.Raft) (m map[string]struct{}) {
 	return m
 }
 
+func ConfigureAPI(api *operations.WeaviateAPI) http.Handler {
+	return configureAPI(api)
+}
+
```

Comment:
I think that this method is not used anywhere, can we remove it?

Reply:
It's used for libweaviate... or will be shortly :D

Reply:
then can we add this method when libweaviate will be ready?

Reply:
Libweaviate is ready now

Reply:
so can we make this change as part of the PR where you will introduce libweaviate? now this method will be just unused

Reply:
Libweaviate is already introduced and available.

Reply:
ok, but where is this method invoked? can you point me to that code where it's used?

Reply:
It's invoked here: https://github.com/weaviate/weaviate/pull/6012/files#diff-445116a3709a8ebebd18c85546b98927f5438b92458fa06bef4fda691ef1ac6fR66